sig/datadog/tracing/contrib/graphql/ext.rbs

```diff
@@ -9,6 +9,8 @@ module Datadog
 
           ENV_ANALYTICS_SAMPLE_RATE: "DD_TRACE_GRAPHQL_ANALYTICS_SAMPLE_RATE"
 
+          ENV_WITH_UNIFIED_TRACER: string
+          EVENT_QUERY_ERROR: String
```

Comment:
Minor: I like it when we inline the actual strings here because we usually use the constants everywhere and it seems slightly weird to me if there isn't a test that fails if we rename the constants. Having the values here is equivalent to having a test, and means we need to change them in two places and thus that seems like a deliberate, rather than an accidental thing :)

```suggestion
          ENV_WITH_UNIFIED_TRACER: "DD_TRACE_GRAPHQL_WITH_UNIFIED_TRACER"
          EVENT_QUERY_ERROR: "dd.graphql.query.error"
```

Reply:
I never though of having rbs as some sort of test for constants...
I'll update the signatures.